freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -60,88 +60,18 @@ public FreeIpaClient(JsonRpcHttpClient jsonRpcHttpClient, String apiVersion) {
         return invoke("user_add", flags, params, User.class);
     }
 
-    // TODO update password expiration aand unpack response
-    //Request: {
-    //    "id": 0,
-    //    "method": "user_mod/1",
-    //    "params": [
-    //        [
-    //            "dhan"
-    //        ],
-    //        {
-    //            "krbpasswordexpiration": {
-    //                "__datetime__": "20380101000000Z"
-    //            },
-    //            "userpassword": "adminadmin1",
-    //            "version": "2.230"
-    //        }
-    //    ]
-    //}
-    //ipa: INFO: Response: {
-    //    "error": null,
-    //    "id": 0,
-    //    "principal": "admin@IPATEST.LOCAL",
-    //    "result": {
-    //        "result": {
-    //            "gidnumber": [
-    //                "1866200061"
-    //            ],
-    //            "givenname": [
-    //                "David"
-    //            ],
-    //            "has_keytab": true,
-    //            "has_password": true,
-    //            "homedirectory": [
-    //                "/home/dhan"
-    //            ],
-    //            "krbcanonicalname": [
-    //                "dhan@IPATEST.LOCAL"
-    //            ],
-    //            "krbpasswordexpiration": [
-    //                {
-    //                    "__datetime__": "20190520200340Z"
-    //                }
-    //            ],
-    //            "krbprincipalname": [
-    //                "dhan@IPATEST.LOCAL"
-    //            ],
-    //            "loginshell": [
-    //                "/bin/sh"
-    //            ],
-    //            "mail": [
-    //                "dhan@ipatest.local"
-    //            ],
-    //            "memberof_group": [
-    //                "ipausers",
-    //                "grp1"
-    //            ],
-    //            "nsaccountlock": false,
-    //            "sn": [
-    //                "Han"
-    //            ],
-    //            "uid": [
-    //                "dhan"
-    //            ],
-    //            "uidnumber": [
-    //                "1866200061"
-    //            ]
-    //        },
-    //        "summary": "Modified user \"dhan\"",
-    //        "value": "dhan"
-    //    },
-    //    "version": "4.6.4"
-    //}
-    public RPCResponse<Object> userSetPassword(String user, String password) throws FreeIpaClientException {
+    public void userSetPassword(String user, String password) throws FreeIpaClientException {
```

Comment:
if userMod returns the RPCResponse<User>, I think this should return it too

Reply:
I'm actually moving in the opposite direction. The caller of the FreeIpaClient doesn't care about the RPSResponse stuff. I intend to remove the RPCResponses from the method signatures and only return the relevant information